### PR TITLE
hashing done, nossq page added

### PR DESCRIPTION
**Pull request by vsaurabh04**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -95,6 +96,11 @@ app.get('/525299210b55528ffbd19b7a50a4ef386e208f9d', function(req,res,next){
   res.render('brochure');
 });
 
+//Rules and regulation 
+app.get('/bc0288e7b5aa3d669ee789fd14af95a8', function(req,res,next){
```
Comment (Vishal260700) [inline app.js]: Why do we need another brochure rendering code while we already have one

File: db1.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "03a9e818926bec9f1c037a354daa1c08": {
```
Comment (Vishal260700) [inline db1.json]: delete all these, they are from db.json

File: public/pronights/assets/css/main.css [hunk 3]
```diff
@@ -1945,7 +1945,7 @@
 	button,
 	.button {
 		background-color: #041625;
-		color: #ffffff ;
+		color: #ffffff !important;
```
Comment (Vishal260700) [inline public/pronights/assets/css/main.css]: this file was changed by shayan hence conflicts with the css and views of the pronights page

File: public/udg_champ/css/main.css [hunk 4]
```diff
@@ -2026,7 +2026,7 @@ ul.icons {
     min-height: 30vh; }
   @media screen and (max-width: 1280px) {
     #banner video {
-      /* display: none; */} }
+      display: none; } }
```
Comment (Vishal260700) [inline public/udg_champ/css/main.css]: changed by shayan

File: views/gallery.ejs [hunk 5]
```diff
@@ -93,9 +93,9 @@
 											<a href="gallery/images/fulls/07.jpg"><img src="gallery/images/thumbs/07.jpg" alt="" title="This right here is a caption." /></a>
 										</div>
 									</div>
-									<!-- <footer>
+									<footer>
```
Comment (Vishal260700) [inline views/gallery.ejs]: changed by shayan

File: views/nossq.ejs [hunk 6]
```diff
@@ -38,8 +38,9 @@
 				<h2>Udghosh Presents To You NOSSQ</h2>
 				<p>A quiz competition among school students all across India.</p>
 				<ul class="actions">
-					<li><a href="/f62045a5685c29dcc61a2cafae030a68e1389db3" class="button big special">Register Now</a></li>
-					<li><a href="/f62045a5685c29dcc61a2cafae030a68e1389db3" class="button big special">Rules and Regulations</a></li>
+					<li><a href="/6d932840157263669f6f378fa14ee190" class="button big special">Register Now</a></li>
+				<!-- to add the rules and regulation put /bc0288e7b5aa3d669ee789fd14af95a8 below and render the rules file in app.js-->
+					<li><a href="f62045a5685c29dcc61a2cafae030a68e1389db" class="button big special">Rules and Regulations</a></li>
```
Comment (Vishal260700) [inline views/nossq.ejs]: href should be /f62045a5685c29dcc61a2cafae030a68e1389db3 for 404.ejs

File: views/pronights.ejs [hunk 7]
```diff
@@ -357,62 +357,44 @@
...
-								<div class="copyright" style="color: white;">
-									&copy; Udghosh'19
-								</div>
-							</div> -->
-							<section id="contact" style = "background-color:black">
-								<!-- Social -->
-									<div class="social column">		
-									
-									<div class="mapouter"><div class="gmap_canvas"><iframe width="100%" height="500" id="gmap_canvas" src="https://maps.google.com/maps?q=indian%20institute%20of%20technology%20kanpur%20&t=k&z=17&ie=UTF8&iwloc=&output=embed" frameborder="0" scrolling="no" marginheight="0" marginwidth="0"></iframe>Google Maps Generator by <a href="https://www.embedgooglemap.net">embedgooglemap.net</a></div><style>.mapouter{position:relative;text-align:right;height:500px;width:90%;margin-left: 5%;margin-right: 5%;margin-bottom: 4%;margin-top: 4%;}.gmap_canvas {overflow:hidden;background:none!important;height:500px;width:100%;}</style></div>
+		<footer id="footer">
```
Comment (Vishal260700) [inline views/pronights.ejs]: changed by shayan, Use the latest commit one for removing shayan changes or do it manually they were minor

File: views/team.ejs [hunk 8]
```diff
@@ -117,6 +117,26 @@
         <!-- End: Header -->
       </div>
       <div class="container">
+        <!-- Start: Desc -->
```
Comment (Vishal260700) [inline views/team.ejs]: remove this section

File: views/team.ejs [hunk 9]
```diff
@@ -105,7 +105,7 @@
           <div class="col-md-7">
             <img src="team/img/logo.png" class="logo">
             <h1>Attend the most awaited Sports Fest</h1>
-            <!-- <h3>to start you up with your Carrer!</h3> -->
+            <h3>to start you up with your Carrer!</h3>
```
Comment (Vishal260700) [inline views/team.ejs]: remove this section by shayan

**Comment by vsaurabh04** [comment]
Check new  pull request

**Pull request closed without merging**